de.json

```diff
@@ -65,7 +65,7 @@
         "Dash.Screens.Settings" : "Einstellungen",
         "Dash.Screens.Exit" : "Beenden",
 
-        "Dash.Actions.PasteFromClipboard" : "Einfügen von Inhalten aus der Zwischenablage",
+        "Dash.Actions.PasteFromClipboard" : "Inhalt der Zwischenablage einfügen",
```

Comment:
My recent commit is shortened as "Zwischenablage einfügen". We should go for shorter sentences where we can.